### PR TITLE
Update gradle-avro dependency version and repo

### PR DESCRIPTION
**Pull request by jeffmarcom**
https://github.com/davidmc24/gradle-avro-plugin/blob/master/CHANGES.md#pre-10-versions

### CODE REVIEW

File: troubleshooting/state-migration/exercise/build.gradle [hunk 1]
```diff
@@ -17,21 +17,21 @@
...
 apply plugin: 'application'
 
 ext.javaExerciseClassName = 'com.ververica.flink.training.exercises.avro.StateMigrationJob'
 
 mainClassName = ext.javaExerciseClassName
 
 dependencies {
-    implementation "org.apache.flink:flink-avro:${flinkVersion}"
-
+    implementation "org.apache.avro:avro:1.9.1"
```
Comment (qinjunjerry) [inline troubleshooting/state-migration/exercise/build.gradle:35]: I am not sure whether we can remove flink-avro...

Comment (jeffmarcom) [reply troubleshooting/state-migration/exercise/build.gradle]: good catch. Fixed

File: troubleshooting/state-migration/exercise/build.gradle [hunk 2]
```diff
@@ -17,13 +17,14 @@
 
 buildscript {
     repositories {
-        jcenter()
+        gradlePluginPortal()
+        mavenCentral()
```
Comment (qinjunjerry) [inline troubleshooting/state-migration/exercise/build.gradle:21]: For further references, can you add a sentence why this replacement is needed?

Comment (jeffmarcom) [reply troubleshooting/state-migration/exercise/build.gradle:21]: updated commit message. Thanks!

**Review by qinjunjerry — APPROVED** [review]
LGTM

**Pull request merged**